pkg/rook/client/client.go

```diff
@@ -43,16 +43,18 @@ type RookRestClient interface {
 	CreateFilesystem(model.FilesystemRequest) (string, error)
 	DeleteFilesystem(model.FilesystemRequest) (string, error)
 	GetStatusDetails() (model.StatusDetails, error)
+	GetObjectStores() ([]model.ObjectStoreResponse, error)
 	CreateObjectStore(store model.ObjectStore) (string, error)
-	GetObjectStoreConnectionInfo() (*model.ObjectStoreConnectInfo, error)
-	ListBuckets() ([]model.ObjectBucket, error)
-	GetBucket(string) (*model.ObjectBucket, error)
-	DeleteBucket(string, bool) error
-	ListObjectUsers() ([]model.ObjectUser, error)
-	GetObjectUser(string) (*model.ObjectUser, error)
-	CreateObjectUser(model.ObjectUser) (*model.ObjectUser, error)
-	UpdateObjectUser(model.ObjectUser) (*model.ObjectUser, error)
-	DeleteObjectUser(string) error
+	DeleteObjectStore(name string) error
+	GetObjectStoreConnectionInfo(name string) (*model.ObjectStoreConnectInfo, error)
+	ListBuckets(name string) ([]model.ObjectBucket, error)
```

Comment:
Perhaps a more descriptive name here like objectStoreName?

Reply:
agreed, i'll make it `storeName`